src/components/Pages/Location/LocationInfo.js

```diff
@@ -152,10 +159,14 @@ const LocationPageFacilityHours = ({ hours }) => {
           content={HoursText}
         />
         {!!hours.exceptions && (
-          <LocationPageBlock
-            title={intl.formatMessage(i18nContact.exceptions)}
-            content={hours.exceptions}
-          />
+          <div className="coa-LocationPage__sub-section-block">
+            <div className="coa-LocationPage__sub-section-block-title-padded">
+              {intl.formatMessage(i18nContact.exceptions)}
+            </div>
+            <div className="coa-LocationPage__sub-section-block-contents">
+              {hours.exceptions}
+            </div>
+          </div>
```

Comment:
The `LocationPageBlock` was a bit too restrictive for what we needed to do here. So, rather than put some conditionals there, i just modified it here as part of the specific component.